<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2339</width>
    <height>1144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1940</x>
      <y>580</y>
      <width>381</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:red</string>
    </property>
    <property name="text">
     <string>Panic</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>1940</x>
      <y>10</y>
      <width>381</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Operate</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>101</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>101</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Forward</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>101</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Reverse</string>
      </property>
     </widget>
     <widget class="QSlider" name="volumeSlider">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>21</width>
        <height>160</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="lightSlider">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>21</width>
        <height>160</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>180</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Volume</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>180</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Light</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="goWaypoint_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>101</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Goto waypoint</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Learn</string>
     </attribute>
     <widget class="QPushButton" name="learntrack_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Learn</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Tracks</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deletetrack_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_button">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>500</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QListWidget" name="track_list">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>256</width>
        <height>80</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Debug</string>
     </attribute>
     <widget class="QComboBox" name="debugImages">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>221</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2339</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
